samba.advisories.yaml

```diff
@@ -93,6 +109,10 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2025-01-03T21:25:03Z
+        type: true-positive-determination
+        data:
+          note: 'Issue has been found in all 4.x versions of samba, no disscussion yet on planned fixed. '
```

Comment:
How about "Upstream has no plan to resolve." or has chosen not to resolve? I'd like to be clear on who has not had the discussion.